cluster/manifests/skipper/configmap-open-policy-agent.yaml

```diff
@@ -0,0 +1,41 @@
+{{ if eq .ConfigItems.skipper_open_policy_agent_enabled "true" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    application: skipper-ingress
+    component: ingress
+  name: open-policy-agent-config
+  namespace: kube-system
+data:
+  opaconfig.yaml: |-
+    discovery:
+      name: discovery
+      resource: v1/bundles/zalando-apps/{{ `{{ .bundlename }}` }}/discovery.tgz
+      service: styra-bundles
+    labels:
+      environment: {{ .Cluster.Environment }}
+      kubernetes-cluster-alias: {{ .Cluster.Alias }}
+      aws-account-id: "{{ accountID .Cluster.InfrastructureAccount }}"
+    services:
+    - credentials:
+        bearer:
+          token: "{{ `{{ .Env.STYRA_TOKEN }}` }}"
```

Comment:
Since this is a secret value it should ideally be stored in a `Secret` object and not it a `ConfigMap`.